addons/web/static/src/core/hotkeys/hotkey_service.js

```diff
@@ -164,7 +164,7 @@ export const hotkeyService = {
             for (const el of getVisibleElements(ui.activeElement, "[data-hotkey]:not(:disabled)")) {
                 const hotkey = el.dataset.hotkey;
                 const overlay = document.createElement("div");
-                overlay.className = "o_web_hotkey_overlay";
+                overlay.className = "o_web_hotkey_overlay position-absolute d-flex justify-content-center align-items-center h-100 h6 text-monospace text-white";
```

Comment:
I'm testing on runbot an size behave weirdly (visual result is ok, but still..)

`h-100` should **not** be necessary thanks to `position-absolute` and  `inset: 0`... but it seems needed... while you removed `width:100%` and everything is fine.. what I'm missing?

Reply:
The issue comes from the margin-bottom of the `h6` class. I replaced the `h-100` by a `m-0` class.

Reply:
Perfect, thanks